test-case/multiple-pipeline.sh

```diff
@@ -149,13 +149,10 @@ do
             die "Wrong -f argument $f_arg, see -h"
     esac
 
-    dlogi "sleep ${OPT_VALUE_lst['w']}s for sound device wakeup"
-    sleep ${OPT_VALUE_lst['w']}
```

Comment:
I think this sleep is used to avoid delay in PM resume.
Not sure if this is still a blocker for us.
@aiChaoSONG @keqiaozhang Any idea？

Reply:
ping?